src/compose.js

```diff
@@ -0,0 +1,3 @@
+import flowRight from 'lodash.flowright';
```

Comment:
how about `lodash.assign`? It uses Object.assign if it's available. and it's faster than extend or merge imo. @oliviertassinari Thoughts?